client/src/components/RoadmapCalendar/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useState } from 'react';
 import S from '@/components/RoadmapCalendar/style';
-import { getRangeFromDate, getYMD, makeDayRow } from '@/lib/utils/date';
+import { getRangeFromDate, getYMD, isSameDay, makeDayRow } from '@/lib/utils/date';
 import RoadmapBars from '@/components/RoadmapBars';
 
 const WEEK_OFFSET = 14;
 
 const RoadmapCalendar = () => {
   const [date, setDate] = useState(new Date());
+  const today = new Date();
+  const isToday = isSameDay(today, date);
```

Comment:
```suggestion
  const isToday = isSameDay(new Date(), date);
```
이렇게 한 줄로 줄여도 되지 않을까요? 아래에서 today는 쓰이지 않는 것 같아서요